### DISCUSSION TITLE
Camwhores completely broken now

### DISCUSSION THREAD

**Discussion by Ronar000** [aclap-dev/video-downloadhelper#971] [Help and Q&A]
### Your question

Can't download a single video from that site. Is it just me or is it an issue with the site?

### Addon & CoApp version and details

Version 9.0.2.12 (stable), target mozilla, locale en-US.
Companion App installed (v2.0.19)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm going to guess we're talking about a totally NSFW site here.  So let's be careful to keep this discussion safe for work.  But that should not prevent you from providing more details.  If this is not a site that requires a registration & a login, you can post URLs here for me to test.  But you need to hide the URLs in images instead of just posting them in plain text.  If the site used to work fine with VDH & now does not, it's possible the site changed something in order to thwart downloading.  I wouldn't be able to compare with how it worked before since I've never visited the site before.  I've actually never heard of it, either.  But if you give me some URLs of specific web pages where there are . . . I'm going to guess they're livestreams . . . I can try to investigate what's going on now.

**Reply by Ronar000** [reply]
![image](https://github.com/aclap-dev/video-downloadhelper/assets/174671966/d3b4f927-0002-4ba9-89b9-acff316b9199)

here is a link to a video that is public. So you wouldn't need to create an account. Yes, VDH used to work just a couple of days ago as well. But I couldn't download anything in the last two days.

Also, apologies. It is an NSFW site. I am not sure if there is a flair or anything to mark the post as NSFW.

**Reply by GrampaWildWilly** [reply]
No need to apologize.  We're all adults here.  As long as we mention NSFW, that should be flag enough.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3810, Firefox 127.0.2 64-bit, licensed VDH 9.0.2.12a1 beta, CoApp 2.0.19.  You didn't mention your operating system or your version of Firefox, so I don't know if those are different between you & me.  But I am running the VDH beta & you're not.  That might make a difference.

I visited your video & VDH just downloaded it.  No issues.  No problems.  I was sort of expecting livestreams but . . . whatever.

But I didn't like your video so I went to the home page of the site & picked something else more up my alley:

![image](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/de9acb4e-0670-4130-8de4-48b6b98d5071)

That one also downloaded without issue.  Maybe you ought to be running the beta.  Try that & see if it makes any difference for you.  As a standard practice, everybody should be running the beta.  Unlike with a lot of other software, the VDH betas are bug corrections, not experimental pieces of unreliable test software.  People really ought to get the beta before they report any problem.  You never know what minor bugs might be corrected.  It looks like we're hitting just such a case here.

**Reply by Ronar000** [reply]
I think I finally figured it out. I think they banned/blacklisted my device IP address. Tried the website with VPN and it suddenly started working again. So I had to manually assign a different IP for my device and it started working again. VDH is working again as well. So I guess it was the site blocking the download.

Appreciate your help with this issue.

**Reply by GrampaWildWilly** [reply]
Interesting.  I am also reminded of the many reports of problems in here that are solved by permitting all the executables in the CoApp directory through both the antivirus & the firewall.  I'd be curious to know if your AV/FW are doing anything to interfere.  Going to a VPN might circumvent some of that.  I'm not too sure about all that because I'm using the AV/FW built into W11.  My experience with VPNs has been singularly unproductive.  It seems the sites I want to access know the IP addresses of the local points of presence of the VPNs & they block those IPs same as they otherwise would geoblock me.

**Reply by Ronar000** [reply]
Sounds like I spoke too soon. The site blocked the new IP as well. Clearly it is an issue with the site. Seems like they are blocking any IP using extensions to download their videos. I can download fine using VDH from other sites. Ah well.

**Reply by GrampaWildWilly** [reply]
But I downloaded a couple of things just fine.  I just went back there again & got something else.  No problem.  It's worth it to eliminate antivirus & firewall issues.  Check those to make sure.

**Reply by GrampaWildWilly** [reply]
How exactly are you determining that the site blocked the IP address?

**Reply by GrampaWildWilly** [reply]
Your success with VDH on other sites would suggest you don't have firewall issues.  But it's possible the firewall is interfering on that particular site for some reason.  I'm just guessing here.

**Reply by Ronar000** [reply]
So it's not that the site is just preventing me using VDH, it is also giving me HTML5 error. None of videos load for me anymore. The same this happened before as well. Previously I checked with other browsers as well. All of them had the same issue.

**Reply by Ronar000** [reply]
Checked through the firewall setting. Even added a new rule for both inbound and outbound for that particular site. Nothing has changed.

**Reply by GrampaWildWilly** [reply]
HTML5 error?  Now that's very suspicious.  In my Firefox settings, I have this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/6fe12c3a-6b33-4a41-8668-33ee762bc909)

I think the Cisco plugin has something to do with HTML5.  Maybe.  Not 100% certain.  But I thought HTML5 support was just built into the browser.  If that's broken, I'd be seriously thinking about reinstalling the browser.  First, I would use the Firefox sync feature to save things you want to be able to reinstate, like bookmarks.  Then I would download the Firefox installer for your environment.  I discussed that with pictures over here: #882.  Then I would get Revo Uninstaller.  It's free for home users.  It does a more thorough job of uninstalling software than most applications' own uninstallers.  Then Revo uninstall Firefox.  Then reinstall Firefox.  Then sync it.  Then see if your HTML5 issue has gone away.

**Reply by Ronar000** [reply]
I have tried running it on different browsers as well....Chrome, Edge, Brave. They all show the same error. So I am expecting it to be an issue with the site, rather than the browser.

**Reply by GrampaWildWilly** [reply]
But I'm not having any problems.  Mind you, it might be something that shows up only after frequent, repeated use.  I'm not likely to visit that site again except to test something.  I just tested something again & it worked fine.  Again.  I'm going to stop now.  It's working for me.  I don't know why it wouldn't work for you.  I assume you took my advice & installed the VDH beta.  That's what I'm using & it works.
